### PR TITLE
[KIECLOUD-435] - Remove deprecated -released tags from image.yaml files

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by msmagnanijr — APPROVED** [review]

File: dashbuilder/image.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ schema_version: 1
 name: "rhpam-7/rhpam-dashbuilder-rhel8"
 description: "Red Hat Process Automation Manager Standalone Dashbuilder 7.10 container image"
 version: "7.10.0"
-from: "ubi8-minimal:8-released"
+from: "registry.redhat.io/ubi8/ubi-minimal:latest"
```
Comment (Kaitou786) [inline dashbuilder/image.yaml:6]: Why do we have two different images? (`registry.redhat.io/ubi8/ubi-minimal:latest` and `registry.redhat.io/ubi8/ubi:latest`). Is it possible to keep all same, possibly the minimal one if works?

Comment (spolti) [reply dashbuilder/image.yaml:6]: This image is new and we start with minimal, the existing image was kept the ub8 only, but it is on my queue to migrate it to ubi8-minimal.
I'll try to do migrate the images before 7.10. Let's see :)

Comment (spolti) [reply dashbuilder/image.yaml:6]: but for now, we'll not change the other images to the ubi8-minimal.

Comment (Kaitou786) [reply dashbuilder/image.yaml:6]: Ah okay 🙂

**Review by Kaitou786 — APPROVED** [review]

**Review by almope — APPROVED** [review]

**Review by jakubschwan — APPROVED** [review]

**Pull request merged**